packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeHelpers.ts

```diff
@@ -0,0 +1,204 @@
+/*!
```

Comment:
As a general rule, I always recommend avoiding the term "helper" in API / file names. It isn't very descriptive, and encourages grouping things together that don't belong together. 

It might be better to make this file `SharedTreeVisualizer.ts`, and even move the root `visualizeSharedTree` function in here.

Reply:
I decided to leave the `visualizedSharedTree()`inside the `DefaultVisualizers.ts` since I want the file to serve as an entry point / repository of all the DDS visualizers. Interested to know if this would be okay.

Reply:
Yeah, either way is fine. In the future, I was thinking I would split out the visualizers such that each has its own file. But for now, either way seems fine to me.